NeutronStandard/Sniffs/Namespaces/DisallowUnusedImportSniff.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace NeutronStandard\Sniffs\Namespaces;
+
+use NeutronStandard\SniffHelpers;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+class DisallowUnusedImportSniff implements Sniff {
+	public function register() {
+		return [T_USE];
```

Comment:
Instead of hooking on `T_USE`, I would hook on `T_STRING`. If they start by `\`, then it's a fully-qualified name, and we don't want them. Else, everything is OK. How does it sound?